### DISCUSSION TITLE
TSC meeting 4 July 2024

### DISCUSSION THREAD

**Discussion by planetf1** [pq-code-package/tsc#79] [Announcements]
The next meeting is arranged for Thu 4 July 1300 UTC

This is 4th July, and our US colleagues will be celebrating.

Does anyone have any specific topics they would like to bring up this week?

I'm inclined to cancel, but if there's sufficient interest we can go ahead

**Comment by planetf1** [comment]
Proposed Agenda -> https://github.com/pq-code-package/tsc/pull/80

Let's go ahead, even if a short meeting.

**Comment by mbbarbosa** [comment]
Apologies, something came up and I won’t be able to attend.
